spec/features/result_filters/qualifications_spec.rb

```diff
@@ -0,0 +1,110 @@
+require "rails_helper"
+
+feature "Qualifications filter", type: :feature do
+  let(:filter_page) { PageObjects::Page::ResultFilters::Qualification.new }
+  let(:results_page) { PageObjects::Page::Results.new }
+
+  describe "qualification page" do
+    before { filter_page.load }
+
+    it "has the correct title and heading" do
+      expect(page.title).to have_content("Filter by qualification")
+      expect(page).to have_content("What you will get")
+    end
+
+    describe "back link" do
+      before do
+        stub_results_page_request
+      end
+
+      it "navigates back to the results page" do
+        filter_page.load(query: { test: "params" })
+        filter_page.back_link.click
+
+        expect_page_to_be_displayed_with_query(
+          page: results_page,
+          expected_query_params: { "test" => "params" },
+        )
+      end
+    end
+
+    describe "de-selecting an option" do
```

Comment:
both me and @TobyRet having a hard time with this

describe "de-selecting an option" 
it "allows the user to select a qualification"

Reply:
Fixed